core/src/main/java/com/rim/IceField/Shovel.java

```diff
@@ -0,0 +1,12 @@
+package com.rim.IceField;
+
+public class Shovel extends ItemBase {
+    public Shovel(String tag) {
```

Comment:
The shovel's tag here should just be "Shovel" automatically.

Reply:
You are right